cmd/minikube/cmd/config/disable.go

```diff
@@ -38,11 +40,22 @@ var addonsDisableCmd = &cobra.Command{
 		if addon == "heapster" {
 			exit.Message(reason.AddonUnsupported, "The heapster addon is depreciated. please try to disable metrics-server instead")
 		}
-		err := addons.SetAndSave(ClusterFlagValue(), addon, "false")
+		cc, err := config.Load(ClusterFlagValue())
```

Comment:
```suggestion
		_, cc := mustload.Partial(ClusterFlagValue())
```
Since you require the `ClusterConfig` I recommend you use `mustload.Partial` instead as that will handle exiting if the cluster can't be loaded.